helpers/fortio.go

```diff
@@ -30,10 +30,10 @@ func sharedHTTPOptions(opts *models.LoadTestOptions) *fhttp.HTTPOptions {
 	httpOpts.ContentType = ""
 	// httpOpts.Payload = fnet.GeneratePayload(*PayloadFileFlag, *PayloadSizeFlag, *PayloadFlag)
 	// httpOpts.UnixDomainSocket = *unixDomainSocketFlag
-	if false { // *followRedirectsFlag {
-		httpOpts.FollowRedirects = true
-		httpOpts.DisableFastClient = true
-	}
+	// if false { // *followRedirectsFlag {
+	httpOpts.FollowRedirects = true
```

Comment:
@girishranganathan quick check - have you verified that the desired redirect behavior happens with an integration test?

Reply:
Yes